src/frame/sdk/GroupApi.ts

```diff
@@ -86,17 +87,56 @@ export function get(groupID: string) {
     return GroupApiEndpoints.callGroupGetApi(groupID).map(formatDetailedGroupResponse);
 }
 
+/**
+ * Create the error strings for missing users
+ */
+function createErrorForInvalidUserList(userKeysResponse: UserKeyListResponseType, memberList: string[]) {
+    const existingUserIDs = userKeysResponse.result.map(({id}) => id);
+    return memberList.filter((userID) => !existingUserIDs.includes(userID));
+}
+
+/**
+ * Checkes that all members listed in member list are existing members
+ */
+function validateMemberList(memberList: string[]) {
+    return memberList !== undefined && memberList.length > 0
```

Comment:
If the `memberList` parameter can be undefined it should be marked as such in the parameter types. So `(memberList?: string[])`.